app/models/api/enrollment_resources.rb

```diff
@@ -0,0 +1,41 @@
+# Serves 'Class Enrollment' card on 'My Academics' page
+class Api::EnrollmentResources
+  def initialize(user_id)
+    @user = User::Current.new(user_id)
+  end
+
+  def enrollment_term_academic_planner
+    # Asking Marty if we can just move this to the CS Link API
+  end
+
+  def enrollment_term_instruction_type_decks
```

Comment:
This is still a work in progress. After all the various Enrollment related resource classes are completed, I'll be patching them into this.